use glob v5.0.3 to follow symlinks and return realpaths

jldec opened the pull request:
The glob module now has options to return realpaths and follow symlinks. With glob v5.0.3. this feature is usable with node-inspector.

Preloaded scripts in symlinked directories now show up properly in the inspector source tree and breakpoints work, fixing issue #370.

f.w.i.w. i also added a little debug tracing in ScriptFileStorage. e.g. to trace during tests do:

    DEBUG=node-inspector:ScriptFileStorage npm test

Fix #370

bajtos commented on lib/ScriptFileStorage.js:
```diff
@@ -149,17 +150,20 @@ $class.listScripts = function(rootFolder, pattern, callback) {
-          var relativePath = relativeUnixPath.split('/').join(path.sep);
-          return path.join(rootFolder, relativePath);
+        result = result.map(function(unixPath) {
+          return unixPath.split('/').join(path.sep);
```
Are you sure the paths are returned as absolute, even though you specify rootFolder via `options.cwd` and not as `options.root`?

Related: 1c06f8ae6 and https://github.com/isaacs/node-glob/commit/f3105a3cfc12d3c6e0a2c931d8cf7f1a6a1ff49a

jldec replied:
yes - i had to fix the code above to avoid prepending the rootfolder twice (which breaks inspector). i also had a little exchange with isaacs about the realpath returned paths being absolute now (and the match results not). https://github.com/isaacs/node-glob/issues/184

bajtos commented:
Thank you for the pull request, sorry for taking so long to review it.

Can you please rebase it on top of the current master and address my comment above?

jldec commented:
rebased and comment addressed.
I also moved to the latest glob version which has updated absolute-path polyfill.
https://github.com/isaacs/node-glob/commit/855230118851fb7d4a6cd7616851f03bca12ad06
Hopefully this will improve things under windows. (i don't have a test setup for windows)

bajtos commented:
Landed via 5e368e8. I have improved the README text a bit, squashed the commits and added more details to the commit message.

Thank you for the contribution!
